docs/src/content/docs/guides/customization.mdx

```diff
@@ -191,7 +191,7 @@ defineConfig({
 
 Starlight has built-in support for adding links to your social media accounts to the site header via the [`social`](/reference/configuration/#social) option in the Starlight integration.
 
-Currently, links to Bitbucket, Codeberg, CodePen, Discord, Email, Facebook, GitHub, GitLab, Gitter, Instagram, LinkedIn, Mastodon, Microsoft Teams, an RSS feed, Stack Overflow, Telegram, Threads, Twitch, Twitter, X, and Youtube are supported.
+You can find a full list of supported social medias in the [Configuration Reference](/reference/configuration/#social).
```

Comment:
I think I might use wording something like this? They’re not all strictly speaking social media I guess (at least e-mail and RSS feel like they stretch that definition), so something more generic probably makes sense:

```suggestion
You can find a full list of supported link icons in the [Configuration Reference](/reference/configuration/#social).
```